lib/flutter_calendar_carousel.dart

```diff
@@ -210,42 +201,18 @@ class _CalendarState<T> extends State<CalendarCarousel<T>> {
       height: widget.height,
       child: Column(
         children: <Widget>[
-          widget.showHeader
-              ? Container(
-                  margin: widget.headerMargin,
-                  child: DefaultTextStyle(
-                      style: widget.headerTextStyle != null
-                          ? widget.headerTextStyle
-                          : defaultHeaderTextStyle,
-                      child: Row(
-                          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                          children: <Widget>[
-                            widget.showHeaderButton
-                                ? IconButton(
-                                    onPressed: () => _setDate(0),
-                                    icon: Icon(Icons.chevron_left,
-                                        color: widget.iconColor),
-                                  )
-                                : Container(),
-                            widget.headerTitleTouchable
-                                ? FlatButton(
-                                    onPressed:
-                                        widget.onHeaderTitlePressed != null
-                                            ? widget.onHeaderTitlePressed
-                                            : () => _selectDateFromPicker(),
-                                    child: headerText,
-                                  )
-                                : headerText,
-                            widget.showHeaderButton
-                                ? IconButton(
-                                    onPressed: () => _setDate(2),
-                                    icon: Icon(Icons.chevron_right,
-                                        color: widget.iconColor),
-                                  )
-                                : Container(),
-                          ])),
-                )
-              : Container(),
+        	CalendarHeader(
```

Comment:
I tried running `flutter format` here, but it actually changed the whole file. I'll just leave it like this for now